### PR TITLE
Fix short background and remove proper nouns

### LINKED ISSUE #46: Losing background colours on scroll
When scrolling down we lose the background colours.

Screenshot_20220218-081944
Screenshot_20220218-082406

### PR DESCRIPTION
**Pull request by wachunga**
Closes #46

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,347 +44,346 @@
   <body
     x-data="gameState()"
     x-init="initialLoad()"
+    :class="`theme-${settings.theme}`"
     @keyup.document="keyPressed"
     @hashchange.window="page = location.hash"
   >
-    <div class="theme-container" :class="`theme-${settings.theme}`">
```
Comment (wachunga) [inline index.html:54]: hide whitespace to see that I only removed theme-container - wasn't needed after all

File: public/main.css [hunk 2]
```diff
@@ -256,6 +259,8 @@ main {
 }
 
 .key {
+  transition: background 0.5s;
+  transition-delay: 1s;
```
Comment (wachunga) [inline public/main.css:263]: I think it's nicer to delay the keyboard update until the tiles have updated

Comment (canadianveggie) [reply public/main.css:263]: That's a good change.

Comment (canadianveggie) [reply public/main.css:263]: Should it be longer?

**Review by canadianveggie — APPROVED** [review]
Good bye ZAMBIA

**Pull request merged**